Adds a 1 second delay for table slamming and Dakimakuras

Poojawa opened the pull request:
## About The Pull Request

What it says on the tin

## Why It's Good For The Game

You slam your palms down on the table.x3515 can no longer happen within the span of a few ticks.

## Changelog
:cl:Poojawa
fix: fixed an oversight that allowed for massive client lagging.
/:cl:

Poojawa committed:
Adds a 2 second delay for table slamming

deathride58 reviewed (changes requested):
This is way too long of a delay. Half of melee delay would be much more appropriate for harm intent, and 1 decisecond would be appropriate for tableslapping.

Poojawa committed:
reduces cool down to 1 second, I guess.

still better than arbitrarily coding self harm that'll just get me screamed at. :<

Poojawa committed:
gives Daki stuff cooldown too by popular request

deathride58 commented on code/game/objects/items/storage/dakis.dm:
```diff
@@ -22,7 +22,7 @@
 	var/body_choice
 	var/custom_name
-
+	user.setClickCooldown(1 SECONDS)
```
The hotkey for using items in your active hand bypasses click delay. This here won't affect the majority of dakispam instances.
